src/corefx/System.Globalization.Native/icushim.cpp

```diff
@@ -170,6 +170,13 @@ bool FindLibWithMajorMinorSubVersion(int* majorVer, int* minorVer, int* subVer)
     return false;
 }
 
+static int32_t g_icuPresent = 0;
+
+extern "C" int32_t GlobalizationNative_ICUPresent()
+{
+    return g_icuPresent;
```

Comment:
I agree.  Wouldn't that be easily addressed just by having InitializeICUShim first check the config flag?  Seems like the rest would fall out naturally.